### PR TITLE
Implement Navigation

### LINKED ISSUE #7: Implement Navigation
image

* Navigation should be shown on every page
* Get logo from Figma
* Links should also work
* Join now should be a clickable button that becomes slightly darker when hovered over
    * This should ideally be a link to their sign up page for now just link it to their Instagram
    * https://www.instagram.com/ai.atucr

### PR DESCRIPTION
**Pull request by jisubang1001**
- 
Screenshot 2024-06-24 212925

- Added Navigation
- Closes #7

### CODE REVIEW

**Comment by jisubang1001** [comment]
I accidentally left my previous version of code on the bottom of NavBar. Please just ignore them.

File: tailwind.config.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ module.exports = {
   theme: {
     extend: {
       ai: {},
```
Comment (khandrew1) [inline tailwind.config.js]: colors are supposed to be in this object

File: src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <Link href="https://www.instagram.com/ai.atucr" className='px-6 py-1 text-white bg-customColor-buttons rounded-full hover:bg-customColor-buttons_darker'>join now</Link>
+        </div>
+      </div>
+      
+    </nav>
+  );
+};
+
+export default NavBar;
+*/
```
Comment (khandrew1) [inline src/components/NavBar.jsx]: remove all this comments, you can always just make new commits before submitting PRs for review

File: src/components/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+"use client";
+import React, { useState } from "react";
```
Comment (khandrew1) [inline src/components/NavBar.jsx]: do not import React

File: src/components/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import Image from "next/image";
+import Link from "next/link";
+import logo from "@/public/logo.png";
```
Comment (khandrew1) [inline src/components/NavBar.jsx]: do not import images like this, when they are in public they can be referenced such as `/logo.png`
for more information: https://nextjs.org/docs/app/building-your-application/optimizing/static-assets

**Review by khandrew1 — CHANGES_REQUESTED** [review]

**Pull request merged**